tagreader.yaml

```
@@ -141,14 +141,13 @@ pn532:
     - text_sensor.template.publish:
         id: rfid_tag
         state: !lambda 'return x;'
+    # Once ESPHome 1.15.2 or later is released, upgrade this according to
+    # https://github.com/adonno/tagreader/issues/6
     - homeassistant.event:
-        event: esphome.rfid_read
-        data:
-          device_id: $devicename
-        data_template: 
-          tag_id: !lambda 'return x;'
-          
-    - if:      
+      event: esphome.tag_scanned
+      data:
+        tag_id: !lambda 'return x;'
```

Just needs a bit of indentation here

```suggestion
        event: esphome.tag_scanned
        data:
          tag_id: !lambda 'return x;'
```